v2/app/collegevine/App.jsx

```diff
@@ -0,0 +1,1357 @@
+/*global Seadragon2, Pivot, PivotCxmlLoader, PivotNumberPicker, PivotViewer, addEventListener,
+PivotNumber_format, makeElement, addText, hasOwnProperty, PivotDatePicker, PivotSlider*/
+
+// PivotView.js:
+// This file is responsible for initializing PivotViewer instances.
+// It creates HTML elements for the UI surrounding the Viewer, handles
+// user interaction with those elements, and also handles custom events
+// that are raised by the Viewer.
+
+/**
+ * Set up and return a PivotViewer instance, created inside the given container element.
+ * This method is responsible for setting up the title bar, search options, filters pane,
+ * and details pane for the control. Unless you're interested in setting those things up
+ * yourself, you should call this method rather than the Pivot.PivotViewer constructor.
+ * @method init
+ * @param div {HTMLElement} The container element
+ * @param useHistory {bool} Whether to update the URL to represent current filter state
+ * @return {Pivot.PivotViewer}
+ */
+var Pivot_init = (Pivot.init = function(div, useHistory) {
```

Comment:
Copy of `v2/app/PivotView.js` with Prettier formatting, so we don’t have to manually wrangle formatting.